See details in tcl-010-Windows-fix-CFLAGS.patch

diff -rubBaN tk.orig/win/Makefile.in tk/win/Makefile.in
--- tk.orig/win/Makefile.in	2024-08-11 03:17:28.798455347 +0000
+++ tk/win/Makefile.in	2024-08-11 03:19:14.220132049 +0000
@@ -170,7 +170,7 @@
 #CFLAGS		= $(CFLAGS_DEBUG)
 #CFLAGS		= $(CFLAGS_OPTIMIZE)
 #CFLAGS		= $(CFLAGS_DEBUG) $(CFLAGS_OPTIMIZE)
-CFLAGS		= @CFLAGS@ @CFLAGS_DEFAULT@ -D_ATL_XP_TARGETING=1 -D__USE_MINGW_ANSI_STDIO=0
+CFLAGS		= @CFLAGS_DEFAULT@ @CFLAGS@ -D_ATL_XP_TARGETING=1 -D__USE_MINGW_ANSI_STDIO=0

 # Special compiler flags to use when building man2tcl on Windows.
 MAN2TCLFLAGS	= @MAN2TCLFLAGS@
