diff -rubBaN tk.orig/win/Makefile.in tk/win/Makefile.in
--- tk.orig/win/Makefile.in	2024-09-27 10:00:19.711174503 +0000
+++ tk/win/Makefile.in	2024-09-27 10:01:21.651175517 +0000
@@ -174,7 +174,7 @@
 #CFLAGS		= $(CFLAGS_DEBUG)
 #CFLAGS		= $(CFLAGS_OPTIMIZE)
 #CFLAGS		= $(CFLAGS_DEBUG) $(CFLAGS_OPTIMIZE)
-CFLAGS		= @CFLAGS@ @CFLAGS_DEFAULT@ -D__USE_MINGW_ANSI_STDIO=0
+CFLAGS		= @CFLAGS_DEFAULT@ @CFLAGS@ -D__USE_MINGW_ANSI_STDIO=0
 
 # Special compiler flags to use when building man2tcl on Windows.
 MAN2TCLFLAGS	= @MAN2TCLFLAGS@
