diff -rubBaN tk.orig/unix/Makefile.in tk/unix/Makefile.in
--- tk.orig/unix/Makefile.in	2024-09-09 03:34:39.289011814 +0000
+++ tk/unix/Makefile.in	2024-09-09 03:37:58.049003793 +0000
@@ -1110,20 +1110,20 @@
 
 tkPkgConfig.o: $(GENERIC_DIR)/tkPkgConfig.c
 	$(CC) -c $(CC_SWITCHES)					\
-		-DCFG_INSTALL_LIBDIR="\"$(LIB_INSTALL_DIR)\"" \
-		-DCFG_INSTALL_BINDIR="\"$(BIN_INSTALL_DIR)\"" \
-		-DCFG_INSTALL_SCRDIR="\"$(SCRIPT_INSTALL_DIR)\"" \
-		-DCFG_INSTALL_INCDIR="\"$(INCLUDE_INSTALL_DIR)\"" \
-		-DCFG_INSTALL_DOCDIR="\"$(MAN_INSTALL_DIR)\"" \
-		-DCFG_INSTALL_DEMODIR="\"$(DEMO_INSTALL_DIR)\"" \
+		-DCFG_INSTALL_LIBDIR="\"$(COOKIT_ROOT)/lib\"" \
+		-DCFG_INSTALL_BINDIR="\"$(COOKIT_ROOT)/bin\"" \
+		-DCFG_INSTALL_SCRDIR="\"$(TK_LIBRARY)\"" \
+		-DCFG_INSTALL_INCDIR="\"$(COOKIT_ROOT)/include\"" \
+		-DCFG_INSTALL_DOCDIR="\"$(COOKIT_ROOT)/share/man\"" \
+		-DCFG_INSTALL_DEMODIR="\"$(TK_LIBRARY)/demos\"" \
 		\
-		-DCFG_RUNTIME_LIBDIR="\"$(libdir)\"" \
-		-DCFG_RUNTIME_BINDIR="\"$(bindir)\"" \
+		-DCFG_RUNTIME_LIBDIR="\"$(COOKIT_ROOT)/lib\"" \
+		-DCFG_RUNTIME_BINDIR="\"$(COOKIT_ROOT)/bin\"" \
 		-DCFG_RUNTIME_SCRDIR="\"$(TK_LIBRARY)\"" \
-		-DCFG_RUNTIME_INCDIR="\"$(includedir)\"" \
-		-DCFG_RUNTIME_DOCDIR="\"$(mandir)\"" \
+		-DCFG_RUNTIME_INCDIR="\"$(COOKIT_ROOT)/include\"" \
+		-DCFG_RUNTIME_DOCDIR="\"$(COOKIT_ROOT)/share/man\"" \
 		-DCFG_RUNTIME_DLLFILE="\"$(TK_LIB_FILE)\"" \
-		-DCFG_RUNTIME_DEMODIR="\"$(DEMO_INSTALL_DIR)\"" \
+		-DCFG_RUNTIME_DEMODIR="\"$(TK_LIBRARY)/demos\"" \
 		\
 		$(GENERIC_DIR)/tkPkgConfig.c
 
